server/functions/helpers.lua

```diff
@@ -0,0 +1,76 @@
+ATL.RegisterCommand = function(name, description, group, cb, suggestions, rcon)
+  if type(name) == 'table' then
+      for i=1, #name do
+          ATL.RegisterCommand(name[i], description, group, cb, suggestions, rcon)
+      end
+      return
+  end
+  if suggestions and #suggestions > 0 then
+      ATL.Commands[name] = {
+          description = description,
+          group = group,
+          suggestions = suggestions,
+      }
+  end
+
+  RegisterCommand(name, function(source, args)
+      local playerId <const> = source
+      if rcon then
```

Comment:
This can most likely be changed later along with the current job handling.

Reply:
In the future, i will change it